database/migrations/2024_08_07_130909_update_product_properties_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('product_properties', function (Blueprint $table) {
+            $table->foreignId('product_property_group_id')
+                ->nullable()
+                ->after('uuid')
+                ->constrained('product_property_groups')
+                ->nullOnDelete();
```

Comment:
Shouldn't the property be deleted when the group is deleted?

Reply:
For safety reasons I made it nullOnDelete(). If a product has any product property from this group a foreign key constraint error appears. if cascadeOnDelete() we have also add that constraint onto the product_product_property table.